client/my-sites/customer-home/cards/launchpad/keep-building.tsx

```diff
@@ -40,6 +40,17 @@ const LaunchpadKeepBuilding = ( { site }: LaunchpadKeepBuildingProps ): JSX.Elem
 		} );
 	};
 
+	useEffect( () => {
+		checklist?.map( ( task: Task ) => {
+			recordTracksEvent( 'calypso_launchpad_task_view', {
+				checklist_slug: checklistSlug,
+				task_id: task.id,
+				is_completed: task.completed,
+				context: 'customer-home',
+			} );
+		} );
+	}, [] );
```

Comment:
Given that we're duplicating this logic, it really feels like we should be handling this via logic in `CustomerHomeLaunchpad`.